python/tvm/ir/transform.py

```diff
@@ -92,6 +104,9 @@ def __enter__(self):
     def __exit__(self, ptype, value, trace):
         _ffi_transform_api.ExitPassContext(self)
 
+    def override_instruments(self, instruments):
```

Comment:
could you add a docstring here to describe this method

Reply:
Done